assets/src/styles/partials/_tooltip.scss

```diff
@@ -1,63 +1,23 @@
-// Based off http://refills.bourbon.io/components/
-.tooltip-item {
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-line-height: 1.5em !default;
-  $dark-gray: #333 !default;
-  $base-font-color: $dark-gray !default;
-  $tooltip-background: white;
-  $tooltip-color: $base-font-color;
-  $tooltip-width: 16em;
-  $tooltip-shadow: 0 2px 2px silver;
-  $tooltip-distance-from-item: 1em;
+// Modified CSS for USWDS Tooltip component
+.usa-tooltip__body {
+    width: auto;
+    white-space: pre-wrap;
+    width: 40vw;
+	max-width: 450px;
```

Comment:
Nit: Fix whitespace. Should always indent with two spaces. I hadn't used vw for width before. Nice!